editor/src/components/canvas/controls/size-box.tsx

```diff
@@ -232,6 +325,34 @@ const ResizeLines = (props: ResizeLinesProps) => {
         edge={edge}
         color={props.color}
       />
+      {(showLabel || isEdgeDragged) && (
+        <PropertyTargetSelector
+          top={
+            top +
+            (props.direction === 'horizontal'
+              ? edge === 'before' && props.direction === 'horizontal'
+                ? -25
```

Comment:
I think these could be added as consts

Reply:
The common logic here was refactored out in e70865c343b6165cba36aff00f57c14962a961ea.